### ISSUE TITLE
why use KL loss instead of Cross entropy?

### ISSUE THREAD

**Issue by PyJulie** [Newbeeer/Max-MIG#3]

**Comment by Newbeeer** [comment]
Hi,

Sorry for the huge delay. KL loss and cross-entropy loss are essentially the same, and we happened to pick KL loss at that time.
